sys/net/network_layer/fib/fib_addon_ipv6.c

```diff
@@ -0,0 +1,145 @@
+/**
+ * FIB addon implementation - for ipv6 address types
+ *
+ * Copyright (C) 2015 Martin Landsmann <Martin.Landsmann@HAW-Hamburg.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ *
+ * @ingroup fib
+ * @{
+ * @file    fib_addon_ipv6.c
+ * @brief   Functions to manage ipv6 FIB entries
+ * @author  Martin Landsmann <Martin.Landsmann@HAW-Hamburg.de>
+ * @}
+ */
+
+#include <errno.h>
+#include "ng_nettype.h"
+#include "ng_fib/ng_fib_table.h"
+#include "ng_fib/ng_fib_addon_ipv6.h"
+
+#ifndef MODULE_NG_IPV6
+#define NG_NETTYPE_IPV6 (28) /**< is AF_INET6 */
```

Comment:
changed the address type to `ng_nettype_t::NG_NETTYPE_IPV6` and set a define for it if the  module is not loaded/used